.azure-pipelines/pipelines.yml

```diff
@@ -158,6 +158,35 @@ jobs:
           artifactName: docker
         condition: always()
 
+  - job: examples
+    dependsOn: ["docker"]
+    displayName: "Verify examples run as documented"
+    pool:
+      vmImage: "ubuntu-18.04"
+    steps:
+      - task: DownloadBuildArtifacts@0
+        inputs:
+          buildType: current
+          artifactName: "docker"
+          itemPattern: "docker/envoy-docker-images.tar.xz"
+          downloadType: single
+          targetPath: $(Build.StagingDirectory)
+      - bash: |
+          docker load < $(Build.StagingDirectory)/docker/envoy-docker-images.tar.xz
+          images=($(docker image list --format "{{.Repository}}"))
+          tags=($(docker image list --format "{{.Tag}}"))
+          for i in "${!images[@]}"; do
+              if [[ "${images[i]}" =~ "envoy" ]]; then
+                  docker tag ${images[$i]}:${tags[$i]} ${images[$i]}:latest
+              fi
+          done
+          docker images
+          sudo apt-get update -y
+          sudo apt-get install -y -qq --no-install-recommends redis-tools
+          export DOCKER_NO_PULL=1
+          ci/verify_examples.sh
```

Comment:
Can you split this to a bash file?

Reply:
moved to do_ci.sh - lmk if you want it somewhere else